### ISSUE TITLE
Null Array item causes failure

### ISSUE THREAD

**Issue by Feiyizhan** [javers/javers#799]
I have an object with a field that is a `Array`. If the `Array` contains an entry that is null, then I get the following exception:
```
java.lang.IllegalArgumentException: argument should not be null
    at org.javers.common.validation.Validate.argumentsAreNotNull(Validate.java:33)
    at org.javers.core.metamodel.object.GlobalIdFactory.createId(GlobalIdFactory.java:44)
    at org.javers.core.graph.LiveCdoFactory.create(LiveCdoFactory.java:29)
    at org.javers.core.graph.LiveCdoFactory.create(LiveCdoFactory.java:15)
    at org.javers.core.graph.EdgeBuilder$MultiEdgeContainerBuilderFunction.apply(EdgeBuilder.java:72)
    at org.javers.core.metamodel.type.ListType.map(ListType.java:33)
```

This issue is similar to [#676](https://github.com/javers/javers/issues/676)。

**Comment by bartoszwalacik** [comment]
Please provide a failing test case (like these https://github.com/javers/javers/tree/master/javers-core/src/test/groovy/org/javers/core/cases)
Does the error still exists in 5.2.5? We did some improvements in this area.

**Comment by KevinXu2014** [comment]
same problem here, basically Javers.compare(null, something new)

it should say, something new added, but Javers threw 'java.lang.IllegalArgumentException' exception.

**Comment by bartoszwalacik** [comment]
fixed in 5.3.1
